src/pages/DetailPage/DetailPage.js

```diff
@@ -16,34 +16,71 @@ class DetailPage extends Component {
   }
 
   componentDidMount() {
-    fetch("/data/product-info.json", {
+    fetch("http://10.58.1.8:8000/product/1", {
       method: "GET"
     })
       .then(res => res.json())
       .then(res => {
-        this.setState({ productInfo: res.productInfo });
-        this.setState({ averageRating: res.averageRating });
-        this.setState({ productReview: res.REVIEWS });
+        console.log("productinfo", res);
+        this.setState({ productInfo: res.data.product_info });
+        // this.setState({ averageRating: res.average_rating });
+        // this.setState({ productReview: res.REVIEWS });
       });
-  }
 
-  // onAdd = () => {
-  //   this.setState({chosenProduct: });
-  // };
+    fetch("http://10.58.1.8:8000/review?product_id=1", {
```

Comment:
서버 주소는 항상 바뀔 수 있어서 변수로 관리해주세요!
config.js 파일 만드셔서
`export const SERVER = 'http://10.58.1.8:8000'`
이렇게 하고 해당 파일에서 Import 해서 사용해주세요!

```suggestion
    fetch(`${SERVER}/review?product_id=1`, {
```

Reply:
네!